src/samsungApi.ts

```diff
@@ -42,13 +51,62 @@ export class SamsungAPI {
   }
 
   static async setDeviceMode(deviceId, mode, token) {
+    // possible modes: 'auto', 'dry', 'cool', 'heat', 'wind'
     const data = {
       'commands' : [{'capability': 'airConditionerMode', 'command': 'setAirConditionerMode', 'arguments': [mode]}],
     };
 
     await Axios.post(`${HOST}/${deviceId}/commands`, data, this.setToken(token));
   }
 
+  static async getFanMode(deviceId, token) {
+    const {
+      data: { fanMode = { } } = {},
+    } = await Axios.get(`${HOST}/${deviceId}/components/main/capabilities/airConditionerFanMode/status`, this.setToken(token));
+    return fanMode.value;
+  }
+
+  static async setFanMode(deviceId, mode, token) {
+    // possible values: 'auto', 'low', 'medium', 'high', 'turbo'
+    const data = {
+      'commands' : [{'capability': 'airConditionerFanMode', 'command': 'setFanMode', 'arguments': [mode]}],
+    };
+
+    await Axios.post(`${HOST}/${deviceId}/commands`, data, this.setToken(token));
+  }
```

Comment:
Since we are using TS, I think that would be better to create an interface to strict those possible values.

What do you think?

> This comment is valid for all lines that you commented with `possible values`

Reply:
Creating an interface sounds good, like that, too!